homeassistant/components/solarlog/const.py

```diff
@@ -26,14 +28,15 @@
 class SolarLogSensorEntityDescription(SensorEntityDescription):
     """Describes Solarlog sensor entity."""
 
-    factor: float | None = None
+    value: Callable[[float | int], float] = round
 
 
 SENSOR_TYPES: tuple[SolarLogSensorEntityDescription, ...] = (
     SolarLogSensorEntityDescription(
         key="time",
         name="last update",
         device_class=SensorDeviceClass.TIMESTAMP,
+        value=as_local,  # type: ignore[arg-type]
```

Comment:
Don't use type ignore to hide a real concern.
```suggestion
        value=as_local,
```